### PR TITLE
docs: Some documentation improvements around online/offline behaviour.

### PR DESCRIPTION
**Pull request by TheRealAgentK**
### Description :memo:

- Some documentation improvements around online/offline behaviour.
- Changes stem from me actually trying to use the online/offline feature. 

**Type of change**

- [x] This change requires a documentation update

**Updates**

:point_right: Documentation update for now
:point_right: We might want to consider additional changes around when and how cached reports are being sent to RG, but that might be a future issue.

### Author to check :eyeglasses:
- [x] Appropriate documentation written (code comments, internal docs)

### Reviewer to check :heavy_check_mark:
- [ ] Project and all contained modules builds successfully
- [ ] Change has been dev-/reviewer-tested, where possible
- [ ] Unit/UI/Automation/Integration tests provided where applicable
- [ ] Code is written to standards
- [ ] Appropriate documentation written (code comments, internal docs)

### CODE REVIEW

**Review by miquelbeltran — APPROVED** [review]

**Comment by miquelbeltran** [comment]
Following up with the chat on Slack, there are few things that need to be clarified in the README.

1. that `isOffline` must be set to `true` to enable the offline support.
2. The example setting up offline support should have `isOffline: true`.

**Review by sumitramanga — COMMENTED** [review]
Changes look good so far, I agree with Miguel on his feedback!

File: README.md [hunk 1]
```diff
@@ -455,17 +455,19 @@ When creating your Raygun client you need to pass through a cache path
...
-  isOffline: false,
+  isOffline: true,
   offlineStorageOptions: {
     cachePath: 'raygunCache/',
     cacheLimit: 1000 // defaults to 100 errors if you don't set this
   }
 });
 ```
 
+Setting `isOffline: true` tells the provider to start in `offline` mode. It will also trigger initialisations of resp. the connection to the cache as specified in `offlineStorageOptions`.
```
Comment (sumitramanga) [inline README.md:466]: Do you mean `response` instead of `resp`

**Comment by sumitramanga** [comment]
Discussed offline - Kai has made the tweak and merged

**Pull request merged**